assets/css/style.css

```diff
@@ -0,0 +1,646 @@
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: -1;
+}
+
+.heading {
+  flex-direction: column;
+  align-items: flex-start;
+  position: relative;
+  border-bottom-left-radius: 5rem;
+  z-index: 10;
+  background-color: #fff;
+}
+
+.heading__title {
+  color: rgba(9, 30, 66, 1);
+  margin-bottom: 12px;
+}
+
+.heading__desc {
+  color: #344563;
+  font-weight: 400;
+  margin-bottom: 20px;
+}
+
+.heading__connect {
+  font-weight: 500;
+  color: rgba(127, 140, 255, 1);
+  margin-top: 14px;
+  margin-bottom: 14px;
+}
+
+.heading__icons {
+  margin: 0.5rem 0;
+}
+
+.heading__icons__ul {
+  align-items: center;
+  justify-content: space-between;
+  width: 11rem;
+}
+
+/*
+ Tonic style
+*/
+.tonic {
+  display: grid;
+  grid-template-columns: auto auto;
+  background-color: #fff;
+  border-radius: 24px;
+  border: solid 1px rgba(223, 225, 230, 1);
+  align-content: flex-start;
+}
+
+/* .tonic .image {
+} */
+
+.content .title {
+  font-weight: 700;
+  color: #091e42;
+  margin-bottom: 12px;
+}
+
+.content .desc {
+  color: rgba(52, 69, 99, 1);
+  margin-bottom: 24px;
+  font-style: normal;
+  font-weight: 400;
+}
+
+.content .summary {
+  margin-bottom: 24px;
+}
+
+.content .summary span {
+  margin-right: 12px;
+}
+
+.content .summary .dotted {
+  display: inline-block;
+  width: 8px;
+  height: 8px;
+  background-color: rgba(193, 199, 208, 1);
+  border-radius: 9999px;
+}
+
+.content .summary .color {
+  color: rgba(107, 119, 140, 1);
+}
+
+.content .tagging {
+  align-items: center;
+  justify-content: flex-start;
+  width: 100%;
+  margin-bottom: 40px;
+}
+
+.tagging li {
+  padding-right: 5px;
+}
+
+/*
+  about styles
+*/
+.about {
+  background-color: #fff;
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  border-top-right-radius: 5rem;
+  align-content: flex-start;
+  padding: 8rem;
+  margin-bottom: -50px;
+}
+
+.about .casualose .border-bottom {
+  border-bottom: 1px solid #d9dbdf;
+}
+
+.about .casualose .header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  width: 100%;
+  padding: 10px 0;
+}
+
+.about .casualose .header span {
+  font-size: 20px;
+  font-weight: 500;
+  line-height: 24px;
+  color: rgba(52, 69, 99, 1);
+}
+
+/*
+  footer styles
+*/
+
+.footer::before {
+  content: "";
+  position: absolute;
+  background-size: cover;
+  background-position: center;
+  background-repeat: no-repeat;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  z-index: -1;
+}
+
+.footer {
+  flex-direction: column;
+  position: relative;
+  border-top-left-radius: 5rem;
+  z-index: 10;
+  background-color: rgba(96, 112, 255, 1);
+  justify-content: center;
+  align-items: center;
+}
+
+.footer .title {
+  font-size: 40px;
+  font-weight: 700;
+  line-height: 52px;
+  text-align: center;
+  color: white;
+  margin-bottom: 12px;
+}
+
+.footer .desc {
+  font-size: 20px;
+  line-height: 28px;
+  text-align: center;
+  color: rgba(235, 235, 255, 1);
```

Comment:
Please use  the right color given in the Figma template to have good design that matches with the original one 😄